### PR TITLE
Feature#9 create a reusable navbar

### LINKED ISSUE #9: Create a reusable navbar
# REQUIREMENTS
Create a navbar that can be used in all the admins view
Features
- When the admin isn't sing-in the navbar doesnt show nothing else that "Foo Corp" logo
- When the admin is sing-in the navbar show all the admins options

# DESIGN
## Log-out nav
image

## Log-in nav
image

### PR DESCRIPTION
**Pull request by HeikelV**
# Description
I created a navbar for navigation between features, to be used in all views to which the admin has access.

 ## Screenshots
If applicable, include screenshots of the results or screenshots that help to see changes
image

### CODE REVIEW

**Review by alexs — DISMISSED** [review]
Lgtm 👍🏻

File: app/assets/stylesheets/navbar.css.scss [hunk 1]
```diff
@@ -0,0 +1,59 @@
+@import url('https://fonts.googleapis.com/css2?family=Quicksand:wght@400;700&display=swap');
+
+$primaryColor: #AEFEFF;
+$very_pale_cyan: #4FBDBA;
+$very_Dark_Cyan: #072227;
+$normal_Cyan: #35858B;
```
Comment (JuanequeX) [inline app/assets/stylesheets/navbar.css.scss]: This variables can be declared into another file to have a better structure  in this file.

File: app/assets/stylesheets/navbar.css.scss [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+.navbar {
+  clear: both;
+  overflow: auto;
+  background-color: $very_Dark_Cyan;
+  border-radius: 50px;
+  font-family: 'Quicksand', sans-serif;
+  color: white;
```
Comment (JuanequeX) [inline app/assets/stylesheets/navbar.css.scss]: Please, sort this block alphabetically, in VS code can be sort with `ctrl + shift + P` and selec the option `sort lines ascending `

File: app/assets/stylesheets/navbar.css.scss [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+  .logo {
+    margin-left: 20px;
+    font-size: 32px;
+    font-weight: bold;
+    text-align: center;
+    float: left;
+    padding: 10px;
+    justify-content: center;
+    align-items: center;
```
Comment (JuanequeX) [inline app/assets/stylesheets/navbar.css.scss]: also sort this block please!

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -1,18 +1,15 @@
...
+  <meta name="viewport" content="width=device-width,initial-scale=1">
+  <%= csrf_meta_tags %>
+  <%= csp_meta_tag %>
+  <%= stylesheet_link_tag "application", "navbar", "data-turbo-track": "reload" %>
+  <%= javascript_importmap_tags %>
+</head>
+<body>
+ <%= render "layouts/navbar" %>
+ <%= yield %>
+</body>
```
Comment (JuanequeX) [inline app/views/layouts/application.html.erb]: This block need to have an extra space of indentation

**Review by JuanequeX — CHANGES_REQUESTED** [review]

**Review by YilKanda — COMMENTED** [review]
If you can, attend to @JuanequeX 's remarks

File: app/assets/stylesheets/navbar.css.scss (hunk 1 above)
Comment (hev-oxe360) [reply app/assets/stylesheets/navbar.css.scss]: waiting for merge and extract common rules

File: app/assets/stylesheets/navbar.css.scss (hunk 2 above)
Comment (hev-oxe360) [reply app/assets/stylesheets/navbar.css.scss]: @JuanequeX solved!

File: app/assets/stylesheets/navbar.css.scss (hunk 3 above)
Comment (hev-oxe360) [reply app/assets/stylesheets/navbar.css.scss]: @JuanequeX solved

File: app/views/layouts/application.html.erb (hunk 4 above)
Comment (hev-oxe360) [reply app/views/layouts/application.html.erb]: @JuanequeX

File: app/assets/stylesheets/navbar.css.scss (hunk 1 above)
Comment (HeikelV) [reply app/assets/stylesheets/navbar.css.scss]: added import of variables

Comment (HeikelV) [reply app/assets/stylesheets/navbar.css.scss]: added main.css.scss

File: app/assets/stylesheets/sass/main.css.scss [hunk 5]
```diff
@@ -0,0 +1,2 @@
+@import "abstracts/variables";
+@import "abstracts/mixins";
```
Comment (JuanequeX) [inline app/assets/stylesheets/sass/main.css.scss]: Add a blank line here

**Review by JuanequeX — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 6]
```diff
@@ -32,4 +32,6 @@
 
 /config/database.yml
 
-.env
\ No newline at end of file
+.env
+
+/.idea
```
Comment (YilKanda) [inline .gitignore]: Add a blank line

File: app/views/layouts/application.html.erb (hunk 4 above)
Comment (Frcomparan) [reply app/views/layouts/application.html.erb]: Change made

File: app/assets/stylesheets/sass/main.css.scss (hunk 5 above)
Comment (Frcomparan) [reply app/assets/stylesheets/sass/main.css.scss]: Change made

File: .gitignore (hunk 6 above)
Comment (Frcomparan) [reply .gitignore]: Change made

**Review by YilKanda — APPROVED** [review]
LGTM!

**Review by JuanequeX — APPROVED** [review]
LGTM

**Review by dannharo — APPROVED** [review]
lgtm :shipit:

**Pull request merged**